pkg/pod/status_test.go

```diff
@@ -624,7 +715,8 @@ func TestMakeTaskRunStatus(t *testing.T) {
 				},
 			}
 
-			got := MakeTaskRunStatus(tr, pod, v1alpha1.TaskSpec{})
+			logger, _ := logging.NewLogger("", "status")
```

Comment:
thanks for threading the logger through! its kind of a pain but i think it's right way to do it :D